oeps/processes/oep-0056-architectural-advisory-process.rst

```diff
@@ -142,6 +143,8 @@ We see three significant problems with the idea of having such a forum:
 
 - Our community is comprised of many independent organizations.  There are times when our component organizations may want to use the Advisory Process on "internal" decisions as well.  Do we create a separate forum meeting for those decisions?  How do we then ensure that we err on the side of having an open, public forum?  Again, logistically, this is quite difficult.
 
+.. _principles:
+
 Appendix A: Advisory Process Principles
```

Comment:
Though TBH, this seems like the core of the proposal, and so shouldn't be an appendix.

Reply:
I don't feel strongly - decided to pull it out to shorten the core proposal itself after some early feedback.